full-stack-tests/core/src/backend/backend.ts

```diff
@@ -121,6 +122,8 @@ async function init() {
     FullStackTestIpcHandler.register();
   }
 
+  ECSchemaRpcImpl.register();
```

Comment:
I _think_ we should be iterating over the list of Rpcs defined within the RpcInterfaces.ts file...

Reply:
This took me the longest time to resolve.  It appears like all the other RPC implementations are registered explicitly (except the test ones) by IModelHost (line 394).  I didn't want to create a dependency to the rpc package in the backend, hence this decision. What do you think?

Reply:
Ah, I didn't realize we registered all of the others by default. I thought it was only a sub-set. If that's the case, this works with me. resolving.